woocommerce/woocommerce-ios
# Add stats tab basic throttling

## Description

This PR adds 2 improvements on Stats tab sync logic.
1. It prevents triggering new sync unless previous finished.
2. It adds 30 s delay before new refresh can be triggered. This applies only for tab switch update, manual pull-to-refresh will still trigger sync anyway.

## Test

1. Run proxy or add couple of `print("sync")` in `StoreStatsAndTopPerformersViewController` -> `syncStats`/`syncSiteVisitStats`/`syncTopEarnersStats`.
2. Check that stats loaded successfully.
3. Try pull-to-refresh, observe it triggers networking.
4. Try switching to products and back - observe no sync networking (there will be 1 request checking availability - subject for a different PR)
5. Wait >30 s and switch tabs - observe stats networking (12+ API requests)

---

Update release notes:
